nais-prod.yaml

```diff
@@ -80,3 +80,11 @@ spec:
       value: aapen-arbeidsrettetOppfolging-endringPaNyForVeileder-v1-p
     - name: ENDRING_PA_MAL_TOPIC
       value: aapen-arbeidsrettetOppfolging-endringPaMal-v1-p
+  azure:
+    application:
+      enabled: true
+  accessPolicy:
+    inbound:
+      rules:
+        - application: veilarbregistrering
+          namespace: paw
```

Comment:
Det er lagt inn config for dette lenger opp, mangler bare å legge inn inbound reglen for veilarbregistrering